src/components.rs

```diff
@@ -0,0 +1,11 @@
+use specs::prelude::*;
+
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Clone, Copy, Debug)]
```
I think all our components will be Clone-able. I think Transforms should be Copy-able.